<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationMain</class>
 <widget class="QWidget" name="RegistrationMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Registration Form</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_username">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_password">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_confirmPass">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="CB_AccType">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	text-align: center;
}</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtBottom</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Account Type:</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>home.png</normaloff>home.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Potential Pet Owner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Foster Parent</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shelter Owner</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_name">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_bio">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_phoneno">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_address">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="CB_State">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	text-align: center;
}</string>
     </property>
     <item>
      <property name="text">
       <string>State:</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../build-cuddlySys-Desktop-Debug/map.png</normaloff>../../build-cuddlySys-Desktop-Debug/map.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AZ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>IA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>KY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ME</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NJ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ND</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WY</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LE_zipcode">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="CB_Gender">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>Gender:</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../build-cuddlySys-Desktop-Debug/gender.png</normaloff>../../build-cuddlySys-Desktop-Debug/gender.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PB_createAcc">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 40px;
    background-color: #ffff66;
    min-width: 80px;
	min-height: 80px;
}</string>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="QL_registerStatus">
     <property name="font">
      <font>
       <family>Manjari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="PB_back">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
